apps/core/scenes/passengerForm/PassengerForm.js

```diff
@@ -2,22 +2,140 @@
+  nationality: ?string,
+  birthDate: ?Date,
+|};
+
+const genderData = [
+  { label: 'Male', value: 'male' },
+  { label: 'Female', value: 'female' },
+];
+
+const nationalityData = [
+  {
+    label: 'Czechia',
+    value: 'CZ',
+  },
+  {
+    label: 'Croatia',
+    value: 'HR',
+  },
+];
+
+export default class PassengerForm extends React.Component<Props, State> {
+  state = {
+    gender: null,
+    givenName: null,
+    lastName: null,
+    nationality: null,
+    birthDate: null,
+  };
+
+  handleGenderChange = (gender: ?string) => {
+    this.setState({ gender });
+  };
+
+  handleGivenNameChange = (givenName: string) => {
+    this.setState({ givenName });
+  };
+
+  handleLastNameChange = (lastName: string) => {
+    this.setState({ lastName });
+  };
+
+  handleBirthDateChange = (birthDate: ?Date) => {
+    this.setState({ birthDate });
+  };
+
+  handleNationalityChange = (nationality: ?string) => {
+    this.setState({ nationality });
+  };
+
+  render() {
+    return (
+      <ScrollView
+        style={styles.container}
+        contentContainerStyle={styles.containerContent}
+      >
+        <View style={styles.form}>
+          <SegmentedButton
+            segmentsData={genderData}
+            selectedValue={this.state.gender}
+            onValueChange={this.handleGenderChange}
+          />
+          <TextInput
+            onChangeText={this.handleGivenNameChange}
+            label="Given names"
+            autoCorrect={false}
+            type="text"
+            labelContainerStyle={styles.inputLabel}
+          />
+          <TextInput
+            onChangeText={this.handleLastNameChange}
+            label="Last name"
+            autoCorrect={false}
+            type="text"
+            labelContainerStyle={styles.inputLabel}
+          />
+          <DateInput
+            date={this.state.birthDate}
+            defaultDate={new Date('1990-01-01')}
+            dateFormat={'MM/DD/YYYY'}
+            onDateChange={this.handleBirthDateChange}
+            placeholder="Select"
+            label="Date of birth"
+            labelContainerStyle={styles.inputLabel}
+            confirmLabel="OK"
+            cancelLabel="CANCEL"
+          />
+          <Picker
+            selectedValue={this.state.nationality}
+            optionsData={nationalityData}
+            onValueChange={this.handleNationalityChange}
+            placeholder="Select"
+            confirmLabel="OK"
+            label="Nationality"
+            labelContainerStyle={styles.inputLabel}
+          />
+        </View>
       </ScrollView>
-    </View>
-  );
+    );
+  }
 }
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: defaultTokens.backgroundBody,
   },
+  containerContent: {
+    flex: 1,
+    alignItems: 'center',
+  },
+  form: {
+    padding: parseInt(defaultTokens.spaceMedium, 10),
+    backgroundColor: defaultTokens.paletteWhite,
+    width: '100%',
+    web: {
+      maxWidth: 720,
```

Comment:
Btw could we create this value as some importable constant? We need to use it on more pages so we wouldn't have to repeat it.

Reply:
Good point, this one is related to other repeating style values which should be added to tokens.
So for now I just added token related todo comment.

I think in next PR we should at least create something like `design-tokens` inside `packages` folder and store those repeating values there to stop flooding code with todos in repeating cases like this.

Reply:
Good idea. It would be nice to move breakpoints from LandingPage `mediaQueriesConfig.js` to that `design-tokens` package as well maybe

Reply:
@JoseDuSV Could you please create issue for it?